device/core/src/interfaces.ts

```diff
@@ -112,19 +112,21 @@ export interface DeviceClientOptions extends X509 {
    * connecting to some other hub (e.g. an Edge Hub), then you may need to set this to the server cert that the hub uses for TLS.
    */
   ca?: string;
-
   /**
-   * Optional object with options specific to the Mqtt transport
+   * Custom User Agent Option 
+   */
```

Comment:
when we ship we'll find a more verbose, explanatory thing to put here :)

Reply:
Yes.